docs/conf.py

```diff
@@ -56,6 +59,13 @@
 myst_heading_anchors = 4
 nb_execution_mode = "off"
 
+# -- Custom Lexers -----------------------------------------------------------
+
+from lexers import FijiGroovyLexer
```

Comment:
Could we please call it `SciJavaGroovyLexer` instead? It really has nothing to do with Fiji specifically. The `#@` syntax is part of the SciJava scripting subsystem.

Reply:
Done!